prius_crossing/prius_crossing.xml

```diff
@@ -136,10 +136,8 @@ def calculate_steer_lane(self):
 	&quot;&quot;&quot;
 	Proportional Steer based on Lane Detected using Front Camera
 	&quot;&quot;&quot;
-	slopes = self.get_line_slope()
-	print(slopes)
-	error = 1 - abs(slopes[0])
-	p = 0.1 * error
+	slopes = self.get_line_slope() # 0, -1, 1, -2, 2 
+	p = 0
```

Comment:
Disables Proportional controller for steer following the lanes, as lane detections need to be refined.

Reply:
A better algorithm to select the slope of the line, which is being perfectly detected using Hough Lines. and maintain a threshold distance.